### ISSUE TITLE
[Feature Request] Show current credit amount while racing

### ISSUE THREAD

**Issue by RonSpawnson** [aimerror/granturismo#6]
To check current credit amount I need to wait for the end of the race. It'd be interesting if the script optionally tracked current credit balance by parsing it from end of race and keeping it displayed during the race. Another idea might be to track credits per last 60 minutes or total credits gained since start of session. 

I might try getting my hands dirty with this if I get some free time. Otherwise if not figured I'd place the idea here too.

**Comment by jilderthoekstra** [comment]
Yeah I also thought about doing something like that.

**Comment by nicholaicascio** [comment]
https://pypi.org/project/pytesseract/

Been looking into this a bit for OCR although it would mean additional dependencies for users.

**Comment by jilderthoekstra** [comment]
Yeah I know about tesseract I have used it before in some projects. But don't want users have to install this dependency. Especially now since I removed the AHK dependency I can probably use pyinstaller to create a binary. So people can just run an exe file without needing to install these dependencies.
